scripts/docker

```diff
@@ -112,19 +112,26 @@ function usage() {
     exit
 }
 
-case "$(uname -m)" in
-    x86_64)
-        arch_suffix='x86'
-        ;;
-    arm64)
-        arch_suffix='arm64'
-        ;;
-    *)
-        arch_suffix='x86'
-        echo "WARNING: unrecognized system architecture ($(uname -m)), falling back to ${arch_suffix}" >&2
-        ;;
-esac
-export COMPOSE_ARCH="hq-compose-arch-${arch_suffix}.yml"
+# This exists because of M1 Macs
+os_suffix='default'
+if [ "$(uname -m)" == "arm64" ]; then
```

Comment:
This could now have misleading results if someone happens to try an ARM Linux system. I'm trying to think of the nicest way to handle that...

Reply:
Offlined with @biyeun on this.